instrument_control/test/anritsu_test.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding : utf-8 -*-
+"""
+Anritsu MG3692C Signal Generator control test file
+
+Author: Nick Materise, Kyle Thompson
+Date:   220728
+
+"""
+
+import sys
+sys.path.append(r'C:\Users\Lehnert Lab\GitHub\measurement\instrument_control')
```

Comment:
Is there a way to use a relative path here instead of hard coding an absolute path?